terraform/staging/main.tf

```diff
@@ -76,3 +76,13 @@ resource "cloudfoundry_service_instance" "datafiles" {
   service_plan     = data.cloudfoundry_service.s3.service_plans["basic-sandbox"]
   recursive_delete = true
 }
+
+data "cloudfoundry_service" "elasticsearch" {
+  name = "aws-elasticsearch"
+}
+
+resource "cloudfoundry_service_instance" "elasticsearch" {
+  name             = "es-staging"
+  space            = data.cloudfoundry_space.space.id
+  service_plan     = data.cloudfoundry_service.elasticsearch.service_plans["es-dev"]
```

Comment:
need to change the plan on this one

Reply:
@raftmsohani does that need to be done here or in a future ticket?